coderd/database/queries/apikeys.sql

```diff
@@ -85,6 +85,22 @@ DELETE FROM
 WHERE
 	user_id = $1;
 
+-- name: DeleteExpiredAPIKeys :exec
+WITH expired_keys AS (
+	SELECT id
+	FROM api_keys
+	-- expired keys only
+	WHERE expires_at < @before::timestamptz
+	LIMIT @limit_count
+)
+DELETE FROM
+	api_keys
+USING
+	expired_keys
+WHERE
+	api_keys.id = expired_keys.id
+;
```

Comment:
We can simplify this, and we should also return the count deleted and log that like we do for aibridge.

```suggestion
-- name: DeleteExpiredAPIKeys :one
WITH expired_keys AS (
  DELETE FROM api_keys
  WHERE expires_at < @before::timestamptz
  LIMIT @limit_count
  RETURNING id
)
SELECT COUNT(*) FROM expired_keys;
```

Reply:
Added: https://github.com/coder/coder/pull/20863/commits/55986afb2c144284f72d3ad78fcf7e80f545bb6f